hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/Constants.java

```diff
@@ -157,6 +157,10 @@ private Constants() {
       "fs.s3a.connection.ssl.enabled";
   public static final boolean DEFAULT_SECURE_CONNECTIONS = true;
 
+  // allow access to requester pay buckets
+  public static final String ALLOW_REQUESTER_PAYS = "fs.s3a.requester-pays.enabled";
```

Comment:
can we make this `fs.s3a.requester.pays.enabled`; avoids confusion about when to use use - over .?